airbyte-webapp/src/hooks/services/Feature/constants.ts

```diff
@@ -7,4 +7,5 @@ export const defaultFeatures = [
   FeatureItem.AllowSync,
   FeatureItem.AllowUpdateConnectors,
   FeatureItem.AllowUploadCustomImage,
+  FeatureItem.AllowOAuthConnector,
```

Comment:
This is not a default feature and should be removed.

Reply:
updated.